sdk/identity/identity/test/public/node/environmentCredential.spec.ts

```diff
@@ -47,14 +49,14 @@ describe("EnvironmentCredential", function () {
     process.env.AZURE_CLIENT_ID = cachedValues.AZURE_CLIENT_ID;
     process.env.AZURE_CLIENT_SECRET = cachedValues.AZURE_CLIENT_SECRET;
 
-    const credential = new EnvironmentCredential();
+    const credential = new EnvironmentCredential(recorder.configureClientOptions({}));
 
     const token = await credential.getToken(scope);
     assert.ok(token?.token);
     assert.ok(token?.expiresOnTimestamp! > Date.now());
   });
 
-  it("authenticates with a client certificate on the environment variables", async function (this: Context) {
+  it.skip("authenticates with a client certificate on the environment variables", async function (this: Context) {
```

Comment:
why skip?

Reply:
same for a couple below in this file

Reply:
I un-skipped the certificate ones :) thank you! They also now run on playback ✨ the new recorder is fancy